### PR TITLE
fix(*): text on edit dedicated service popup overflows out of modal

### PR DESCRIPTION
**Pull request by radireddy**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

## Title of the Pull Requests 
text shown on update my service popup overflows and goes out of the modal boundary

### Description of the Change
replaced HTML table with HTML version of oui-datagrid

### Benefits
provides responsive content

### Possible Drawbacks
none

### Applicable Issues
MFRWW-691

### CODE REVIEW

File: client/app/account/billing/autoRenew/update/billing-autoRenew-update.html [hunk 1]
```diff
@@ -9,123 +9,116 @@
...
-            </table>
+            <div class="oui-datagrid-responsive-container">
+                <div class="oui-datagrid-responsive-container__sticky-container">
+                    <div class="oui-datagrid-responsive-container__overflow-container">
+                    <table class="oui-datagrid">
+                        <thead class="oui-datagrid__headers">
+                        <tr>
+                            <th class="oui-datagrid__header" tabindex="0" data-translate="autorenew_service_name"></th>
+                            <th class="oui-datagrid__header" tabindex="0" data-translate="autorenew_service_type"></th>
+                            <th class="oui-datagrid__header text-nowrap" tabindex="0" ><span data-translate="autorenew_service_renew"></span></th>
```
Comment (marie-j) [inline client/app/account/billing/autoRenew/update/billing-autoRenew-update.html]: Why not directly use `data-translate` on `<th>` like above

File: client/app/account/billing/autoRenew/update/billing-autoRenew-update.html [hunk 2]
```diff
@@ -9,123 +9,116 @@
...
+                            <th class="oui-datagrid__header text-nowrap" tabindex="0"><span data-translate="autorenew_service_frequency"></span></th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                            <tr class="oui-datagrid__row" data-ng-repeat="service in selectedServices | orderBy:'serviceType':false track by $index">
+                                <td class="oui-datagrid__cell">
+                                    <span data-ng-bind="::service.serviceId | sliceContent:25"></span>
```
Comment (marie-j) [inline client/app/account/billing/autoRenew/update/billing-autoRenew-update.html]: Can't `ng-bind` directly be used on `<td>` ?

File: client/app/account/billing/autoRenew/update/billing-autoRenew-update.html [hunk 3]
```diff
@@ -9,123 +9,116 @@
...
+                                </td>
+                                <td class="oui-datagrid__cell text-center">
+                                    <select name="renewPeriod"
+                                            class="form-control"
+                                            data-ng-if="service.newRenewType === 'auto' && service.possibleRenewPeriod.length > 1"
+                                            data-ng-change="onChange()"
+                                            data-ng-options="period as getPeriodTranslation(period) for period in service.possibleRenewPeriod"
+                                            data-ng-model="service.newRenewPeriod">
+                                    </select>
+                                    <span data-ng-if="service.newRenewType === 'auto' && service.possibleRenewPeriod.length === 1"> {{getPeriodTranslation(service.newRenewPeriod)}}</span>
```
Comment (marie-j) [inline client/app/account/billing/autoRenew/update/billing-autoRenew-update.html]: `data-ng-bind` instead

File: client/app/account/billing/autoRenew/update/billing-autoRenew-update.html [hunk 4]
```diff
@@ -9,123 +9,116 @@
...
+                            <th class="oui-datagrid__header text-nowrap" tabindex="0"><span data-translate="autorenew_service_update_future"></span></th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                            <tr class="oui-datagrid__row" data-ng-repeat="service in selectedServices | orderBy:'serviceType':false track by $index">
+                                <td class="oui-datagrid__cell">
+                                    <span data-ng-bind="service.serviceId | sliceContent:25"></span>
+                                </td>
+                                <td class="oui-datagrid__cell">
+                                    <span data-ng-bind="'autorenew_service_type_' + service.serviceType | translate"></span>
```
Comment (marie-j) [inline client/app/account/billing/autoRenew/update/billing-autoRenew-update.html]: You could directly use `data-translate` here

File: client/app/account/billing/autoRenew/update/billing-autoRenew-update.html [hunk 5]
```diff
@@ -9,123 +9,108 @@
...
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
+            <div class="oui-datagrid-responsive-container">
+                <div class="oui-datagrid-responsive-container__sticky-container">
+                    <div class="oui-datagrid-responsive-container__overflow-container">
+                    <table class="oui-datagrid">
+                        <thead class="oui-datagrid__headers">
+                        <tr>
```
Comment (FredericEspiau) [inline client/app/account/billing/autoRenew/update/billing-autoRenew-update.html]: Indentation

File: client/app/account/billing/autoRenew/update/billing-autoRenew-update.html [hunk 6]
```diff
@@ -9,123 +9,108 @@
...
```
Comment (FredericEspiau) [inline client/app/account/billing/autoRenew/update/billing-autoRenew-update.html]: Indentation

**Comment by ovh-cds** [comment]
CDS Report build-yarn#1282.0 ✘  
* Stage 1
  * deps ✔ 
* Stage 2
  * build ✘  
  * test ✔

**Pull request merged**